### DISCUSSION TITLE
Crazyswarm with Gazebo?

### DISCUSSION THREAD

**Discussion by santiagorg2401** [USC-ACTLab/crazyswarm#386] [Q&A]
Hello,
Do you know if there is a Crazyflie Gazebo simulator compatible with this Crazyswarm project?, or, do you have one?
Thanks in advance.

**Comment by jpreiss** [comment]
Not yet. We discuss ROS-based simulation in #326 and have draft PR in #350, but this is about making the rudimentary simulator from `pycrazyswarm` available as a ROS node.

There is [CrazyS](https://github.com/gsilano/CrazyS), but I don't know anything about its support for Crazyswarm (or real hardware and multiple CFs in general).

Historically, we have not tried very hard to integrate with heavy-duty realistic simulators because they can become slow for large multi-robot projects.

**Comment by santiagorg2401** [comment]
Ok, thank you.
